docs/elasticsearch.rst

```diff
@@ -1,6 +1,56 @@
 ElasticSearch
 =============
 
+Playing Nice with ES: Simple things to know to get reasonable performance
```

Comment:
I was going to say that you could put this https://github.com/dimagi/commcare-hq/blob/master/docs/es_query.rst because it's linked at the bottom of this page, but that doesn't look like a very helpful doc. Maybe replace `Querying ElasticSearch` with this?